lisa/tools/go.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import TYPE_CHECKING, List, Type
+
+from lisa.base_tools.wget import Wget
+from lisa.executable import Tool
+from lisa.operating_system import Posix, Suse
+from lisa.tools.echo import Echo
+from lisa.tools.rm import Rm
+from lisa.tools.tar import Tar
+from lisa.util import LisaException
+
+if TYPE_CHECKING:
+    from lisa.node import Node
+
+
+class Go(Tool):
+    def __init__(self, node: "Node") -> None:
+        super().__init__(node)
+        self._thread_count = 0
+
+    @property
+    def command(self) -> str:
+        return "go"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Wget, Tar, Rm]
+
+    def _install(self) -> bool:
+        if isinstance(self.node.os, Suse):
```

Comment:
again, The logic of Suse is the same as posix, so it doesn't need separate the logic out. Even the whole logic is not needed. Let Windows fails as is, because it's not widely supported in LISA.

Reply:
Removed Suse. Kept only Posix and failing on Windows